docs/source/guide/custom_metric.md

```diff
@@ -98,7 +98,10 @@ Set up a custom agreement metric for a specific project in Label Studio Enterpri
 
 1. Within a project on the Label Studio UI, click **Settings**.
 2. Click **Quality**.
-3. Under **Annotation Agreement**, use the drop-down menu to select **Custom agreement metric**.
+3. Under **Annotation Agreement**:
+   - **Metric name**: Use the drop-down menu to select **Custom agreement metric**.
+   - **Lamda Tags**: Add tags to AWS lambda function using the syntax `{tag_name:tag_value}`.
```

Comment:
It should be mentioned that it's on-premise-only feature. Also providing the link https://docs.aws.amazon.com/lambda/latest/dg/configuration-tags.html